scripts/typos_rollout_check.py

```
@@ -0,0 +1,248 @@
+
+    Returns
+    -------
+    tuple[Path, ...]
+        Sorted repository-relative paths.
+
+    Raises
+    ------
+    subprocess.CalledProcessError
+        If Git cannot list the repository's tracked paths.
+    """
+    raw = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)